pkg/cli/cmd/directory/check.go

```diff
@@ -0,0 +1,134 @@
+package directory
+
+import (
+	"encoding/json"
+	"os"
+
+	"github.com/aserto-dev/clui"
+	"github.com/aserto-dev/go-directory/aserto/directory/reader/v3"
+	"github.com/aserto-dev/topaz/pkg/cli/cc"
+	"github.com/aserto-dev/topaz/pkg/cli/clients"
+	"github.com/aserto-dev/topaz/pkg/cli/jsonx"
+	"github.com/pkg/errors"
+	"google.golang.org/protobuf/encoding/protojson"
+)
+
+type CheckPermissionCmd struct {
```

Comment:
I think we should just expose `Check`, not `CheckPermission` and `CheckRelation`